padle/padle.py

```diff
@@ -64,8 +66,8 @@ async def red_get_data_for_user(self, *, user_id):
         """Get a user's personal data."""
         all_guesses = await self.config.user_from_id(user_id).all_guesses()
         if all_guesses:
-            data = f"You have {len(all_guesses)} days of guess data stored."
-            f"Here are your guesses:\n{json.dumps(all_guesses)}"
+            data = (f"You have {len(all_guesses)} days of guess data stored."
+                    f"Here are your guesses:\n{json.dumps(all_guesses)}")
```

Comment:
Space missing between the two sentences.

Have you looked at the output of this JSON? If you want to prettify it, look into `.dump`'s `indent` argument. Or write a function for displaying it, and do the logic in that function instead of deciding here.